### PR TITLE
Update ES

### PR DESCRIPTION
**Pull request by amekare**
corrections to audio applications

### CODE REVIEW

**Comment by luong-komorebi** [comment]
@claromes 
@alejandro-krieger
@troyodk
Can you guys help me review this PR? SInce I m not literate in ES, your opinions would provide great values here

File: README_es-ES.md [hunk 1]
```diff
@@ -126,15 +126,15 @@
...
-- [Netease Music](http://music.163.com/#/download) - Un reproductor de música para netease - un servicio de música en china.
+- [![Open-Source Software][OSS Icon]](https://about.musixmatch.com/apps/) [MusixMatch](https://about.musixmatch.com/apps/) - Una programa para búsqueda de las letras de canciones. Se puede sincronizar con la música.
+- [Netease Music](http://music.163.com/#/download) - Un reproductor de música para netease - un servicio de música en China.
 - [![Open-Source Software][OSS Icon]](https://github.com/nukeop/nuclear) [Nuclear](http://nuclear.gumblert.tech/) - Basado en Electron, Nuclear es una programa multiplatforma para el reproductor de medios puede utilizar múltiples fuentes para la reproducción.
-- [Ocenaudio](http://www.ocenaudio.com/whatis) - Una programa multiplatforma fácil, es rapido e funcional editor de música. es un buen programa para editar y analizar archivos de música.
+- [Ocenaudio](http://www.ocenaudio.com/whatis) - Una programa multiplatforma fácil, es rápido e funcional editor de música. Es un buen programa para editar y analizar archivos de música.
```
Comment (claromes) [inline README_es-ES.md]: Change "multiplatforma" to "multiplataforma"

Comment (luong-komorebi) [reply README_es-ES.md]: Since the owner leaves this PR hanging for too long, I decided to step it and edited this

**Comment by luong-komorebi** [comment]
Thank you so much @amekare

**Pull request merged**